docs/apv.md

```diff
@@ -249,10 +250,10 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 
 ### Artikel 27 - Overvallen & Oplichten
 
-1. Het overvallen/oplichten van mensen is verboden, dit mag alleen wanneer een persoon bezig is met criminele activiteiten.
-2. Het is in geen enkele situatie toegestaan om geld van mensen hun bank te stelen of om te scammen met voertuigen.
+1. Het overvallen/oplichten/bestelen van mensen is verboden, dit mag alleen wanneer een persoon bezig is met criminele activiteiten.
+2. Het is in geen enkele situatie toegestaan om geld van mensen hun bank te stelen, om te scammen met voertuigen, om spelers te overvallen bij het gemeentehuis of om facturen en/of andere items te spelen van spelers die bezig zijn met een burgerjob.
```

Comment:
Dus als ik niet bij het gemeentehuis ben mag het wel?

Reply:
Jawel, maar dat staat vermeld in lid 1. In lid 2 wordt het expliciet verduidelijkt dat het onder andere NOOIT is toegestaan om mensen te overvallen/oplichten bij het gemeentehuis, omdat daar vrijwel altijd nieuwe spelers staan. In lid 1 staat niet dat ontvoeren ook verboden is, in lid 2 wordt toegevoegd dat het wel verboden is om mensen te ontvoeren bij het gemeentehuis.

Reply:
2. Het is in geen enkele situatie toegestaan om geld van mensen hun bank te stelen, om te scammen met voertuigen, om spelers te overvallen bij het gemeentehuis of om facturen en/of andere items te spelen van spelers die bezig zijn met een burgerjob.

In lid 2 staat in de laatste zin een kleine spellingsfout, het moet zijn en/of andere items te stelen van spelers etc.
Nu staat er bovenstaande.

Groetjes